code/datums/licences/license.dm

```diff
@@ -31,3 +31,9 @@
 	name = "Grandfathered In"
 	url = "https://en.wikipedia.org/wiki/Grandfather_clause"
 	attribution_mandatory = FALSE
+
+/decl/license/good_faith
+	name = "Good Faith Usage"
+	url = "https://en.wikipedia.org/wiki/Good_faith_(law)"
+	attribution_mandatory = FALSE
```

Comment:
I'd say that we do want attribution to be mandatory, especially if we're going with good faith